src/utils/common.h

```diff
@@ -532,7 +532,7 @@ static inline int is_multicast_ether_addr(const u8 *a)
 
 #define broadcast_ether_addr (const u8 *) "\xff\xff\xff\xff\xff\xff"
 
-#include "wpa_debug.h"
+#include <utils/wpa_debug.h>
```

Comment:
if you move from zephyr/src/utils -> zephyr/src then I guess this isn't needed?

Reply:
It is needed. Unfortunately, using `#include "file.h"` type of includes always prefers a local file, so I would still need to change it to `<wpa_debug.h>`.

Reply:
ok, thanks.